tsconfig.json

```diff
@@ -23,5 +23,5 @@
   },
   "include": [
     "src"
-  ]
+, "src/utils/.tsx"  ]
```

Comment:
Why is this line added?

Reply:
not sure, didn't manually add it.
removed in 741d64c, but had to add `downlevelIteration` because we're now using iterators `urlParams.entries()`

Reply:
`downlevelIteration`...  🤔 Hm, interesting. Weird how there isn't a copy support on that object, but still good job 👍